### ISSUE TITLE
Communicate Community Server and Document Server = Converter url: download

### ISSUE THREAD

**Issue by rezoor** [ONLYOFFICE/CommunityServer#190]
Just set up and can not get to work.
Everywhere https.

Version
sd01 = onlyoffice-documentserver-5.4.2  (Work checked on Nextcloud 17.0.2 with app onlyoffice, Maybe it doesn’t mean anything))
sd03 = onlyoffice-communityserver-10.5.1-317

Log on communityserver  (Every time when I try to save the settings)
tail /var/log/onlyoffice/web.files.log -f

2020-01-22 07:44:55,654 ERROR [52] ASC.Files - Download stream header https://10.20.20.117/Products/Files/httphandlers/filehandler.ashx?action=empty&title=.docx&X-REWRITER-URL=https:%2F%2Fds03.example.ex:443 System.Exception: Invalid header
  at ASC.Web.Files.FileHandler.EmptyFile (System.Web.HttpContext context) [0x00056] in <8b43e55685d446ce8107f232575cd85f>:0
2020-01-22 07:44:56,669 ERROR [51] ASC.Files - Download stream header https://10.20.20.117/Products/Files/httphandlers/filehandler.ashx?action=empty&title=.docx&X-REWRITER-URL=https:%2F%2Fds03.example.ex:443 System.Exception: Invalid header
  at ASC.Web.Files.FileHandler.EmptyFile (System.Web.HttpContext context) [0x00056] in <8b43e55685d446ce8107f232575cd85f>:0
2020-01-22 07:44:57,679 ERROR [6] ASC.Files - Download stream header https://10.20.20.117/Products/Files/httphandlers/filehandler.ashx?action=empty&title=.docx&X-REWRITER-URL=https:%2F%2Fds03.example.ex:443 System.Exception: Invalid header
  at ASC.Web.Files.FileHandler.EmptyFile (System.Web.HttpContext context) [0x00056] in <8b43e55685d446ce8107f232575cd85f>:0
2020-01-22 07:44:59,648 ERROR [11] ASC.Files - Converter DocService check error ASC.Web.Core.Files.DocumentService+DocumentServiceException: download
  at ASC.Web.Core.Files.DocumentService+DocumentServiceException.ProcessResponseError (System.String errorCode) [0x000ed] in <b6ff6b2a22de49f287c183e4ab45002a>:0
  at ASC.Web.Core.Files.DocumentService.GetResponseUri (System.String jsonDocumentResponse, System.String& responseUri) [0x00041] in <b6ff6b2a22de49f287c183e4ab45002a>:0
  at ASC.Web.Core.Files.DocumentService.GetConvertedUri (System.String documentConverterUrl, System.String documentUri, System.String fromExtension, System.String toExtension, System.String documentRevisionId, System.String password, System.Boolean isAsync, System.String signatureSecret, System.String& convertedDocumentUri) [0x002e0] in <b6ff6b2a22de49f287c183e4ab45002a>:0
  at ASC.Web.Files.Services.DocumentService.DocumentServiceConnector.CheckDocServiceUrl () [0x00099] in <8b43e55685d446ce8107f232575cd85f>:0
![Untitled01](https://user-images.githubusercontent.com/60135190/72896065-f1572980-3d2f-11ea-877e-b9d398b4872c.jpg)

**Comment by alexanderonlyoffice** [comment]
@rezoor, please specify the following information:

1. Are Community Server and Document Server installed on the same server or on different machines? 
2. Installation type of Community Server and Document Server (docker or deb/rpm);
3. Please make sure you have indicated the correct addresses of the Document Server in two first fields and the address of the Community Server in the last bottom field. According to the screenshot the address of the Community Server is indicated with HTTP protocol. Please note the Community Server and Document Server must be accessible for each other by these addresses.  

There is a possibility to deploy Community Edition that included Community Server and Document Server by our installation script (you may decide during the installation whether to install them with Docker or to choose package installation). The installation script will deploy both components on one server with all configurations done by default. Please follow this [link](https://helpcenter.onlyoffice.com/server/docker/opensource/index.aspx) to find more information.

**Comment by alexanderonlyoffice** [comment]
@rezoor, please specify the following information:

1. Are Community Server and Document Server installed on the same server or on different machines? 
2. Installation type of Community Server and Document Server (docker or deb/rpm);
3. Please make sure you have indicated the correct addresses of the Document Server in two first fields and the address of the Community Server in the last bottom field. According to the screenshot the address of the Community Server is indicated with HTTP protocol. Please note the Community Server and Document Server must be accessible for each other by these addresses.  

There is a possibility to deploy Community Edition that included Community Server and Document Server by our installation script (you may decide during the installation whether to install them with Docker or to choose package installation). The installation script will deploy both components on one server with all configurations done by default. Please follow this [link](https://helpcenter.onlyoffice.com/server/docker/opensource/index.aspx) to find more information.

**Comment by rezoor** [comment]
Thanks for the advice! Through sh immediately everything worked.
But this is not the way of the Jedi)))

I already deleted the machine, but for the history I will describe the process.
The servers were on two different machines through rpm, I used to work and update through repositories and try to bring everything in one line.

Firewal - disabled
SElinux - disabled

wget download everything on both sides
